portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,32 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> messages;
+
+  @Override
+  public void init() {
+    messages = new ArrayList<>();
+    messages.add("First of all... how are you?");
+    messages.add("The worst thing about prison was the dementors.");
+    messages.add("If I dont have some cake soon, I might die.");
+  }  
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String json = convertToJsonUsingGson(messages);
+
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Shandy!</h1>");
+    response.getWriter().println(json);
+  }
+
+  /**
+   * Converts a ServerStats instance into a JSON string using the Gson library. Note: We first added
+   * the Gson library dependency to pom.xml.
```

Comment:
nit: remove this second sentence of this comment?

Reply:
I'll look more carefully at when copying and pasting from other files next time.